soh/soh/Enhancements/debugger/debugSaveEditor.h

```diff
@@ -24,6 +24,7 @@ typedef struct {
 } FlagTable;
 
 // Reference https://tcrf.net/Proto:The_Legend_of_Zelda:_Ocarina_of_Time_Master_Quest/Event_Editor
+// This information predates gigaleak and is considered safe to use
```

Comment:
it'd be nicer to just have the accessed and last modified dates here, don't need to mention gigaleak in the codebase, just the dates will be enough to let us know we're in the clear

Reply:
```suggestion
// The source was last referenced on 9/3/2022 and had a last updated value of 5/2/2020
```
How's that?

Reply:
2022-09-03 and 2020-05-02